### PR TITLE
feat(completions): add basic nushell completion file

### LINKED ISSUE #254: feat:  add nushell completions
after looking at #253 it made me realize that we do not have [nushell](https://github.com/nushell/nushell) completions.

I am surprised it took me this long to realize this, as I actually use nushell. But either way, completions would be great :heart:

### PR DESCRIPTION
**Pull request by gierens**
This adds an initial nushell completion file generated with https://github.com/dscottboggs/nu-completions from our fish completions ... I had to remove a few things to make it work. The script doesn't like `\` and multiline strings. Maybe we can add those manually after verifying that this at least works.

Resolves #254

### CODE REVIEW

**Review by cafkafk — CHANGES_REQUESTED** [review]
you should add pthorpe to the codeowners file for nushell completions

**Review by PThorpe92 — APPROVED** [review]

File: completions/nush/eza.nu
Comment (cafkafk) [inline completions/nush/eza.nu:49]: We should probably align the comments

Comment (PThorpe92) [reply completions/nush/eza.nu:49]: image
it auto aligns in your terminal when you use them. but yeah probably should still

Comment (gierens) [reply completions/nush/eza.nu:49]: that's a good point ... thought about that too

**Comment by daviessm** [comment]
It might be a fun task for someone to add nushell to https://crates.io/crates/clap_complete so we can generate them all automatically once `clap` is done.

**Comment by gierens** [comment]
> It might be a fun task for someone to add nushell to https://crates.io/crates/clap_complete so we can generate them all automatically once `clap` is done.
> 
> Edit: it already exists - https://crates.io/crates/clap_complete_nushell

Uh, we definitely should check out claps extesions once it's merged.

**Comment by daviessm** [comment]
> Uh, we definitely should check out clap's extesions once it's merged.

Yeah I'm hoping to do that off the back of #400

**Review by cafkafk — APPROVED** [review]
LGTM :D

**Pull request merged**
